tests/EmissionTestLIDOLM.t.sol

```diff
@@ -0,0 +1,182 @@
+  //  asset The asset on which to put reward on, usually Aave aTokens or vTokens (variable debt tokens)
+  //  emission Total emission of a `reward` token during the whole distribution duration defined
+  // E.g. With an emission of 10_000 MATICX tokens during 1 month, an emission of 50% for variableDebtPolWMATIC would be
+  // 10_000 * 1e18 * 50% / 30 days in seconds = 1_000 * 1e18 / 2_592_000 = ~ 0.0003858 * 1e18 MATICX per second
+
+  address constant wETHLIDO_A_Token = 0xfA1fDbBD71B0aA16162D76914d69cD8CB3Ef92da;// TODO: hardcoded for now will use lib when address book is updated
+  address constant wETH_ORACLE = AaveV3EthereumAssets.WETH_ORACLE;
+
+
+  struct EmissionPerAsset {
+    address asset;
+    uint256 emission;
+  }
+
+  address constant EMISSION_ADMIN = 0xac140648435d03f784879cd789130F22Ef588Fcd; // ACI
+  address constant REWARD_ASSET = wETHLIDO_A_Token;
+
+  IEACAggregatorProxy constant REWARD_ORACLE = IEACAggregatorProxy(wETH_ORACLE);
+
+  ITransferStrategyBase constant TRANSFER_STRATEGY =
+    ITransferStrategyBase(0x4fDB95C607EDe09A548F60685b56C034992B194a); // new deployed strategy
+
+  uint256 constant TOTAL_DISTRIBUTION = 80 ether; // 80 awETH/14 Days
+  uint88 constant DURATION_DISTRIBUTION = 14 days;
+  
+  // Not needed as ACI is first LP in market
+  // address wETHLIDO_WHALE = 0xac140648435d03f784879cd789130F22Ef588Fcd;
+  address WETH_A_TOKEN_WHALE = 0x464C71f6c2F760DdA6093dCB91C24c39e5d6e18c; // collector
+
+  function setUp() public {
+    vm.createSelectFork(vm.rpcUrl('mainnet'), 20611541); // change this when ready
+  }
+
+  function test_activation() public {
+    vm.startPrank(EMISSION_ADMIN);
+    /// @dev IMPORTANT!!
+    /// The emissions admin should have REWARD_ASSET funds, and have approved the TOTAL_DISTRIBUTION
+    /// amount to the transfer strategy. If not, REWARDS WILL ACCRUE FINE AFTER `configureAssets()`, BUT THEY
+    /// WILL NOT BE CLAIMABLE UNTIL THERE IS FUNDS AND ALLOWANCE.
+    /// It is possible to approve less than TOTAL_DISTRIBUTION and doing it progressively over time as users
+    /// accrue more, but that is a decision of the emission's admin
+    IERC20(REWARD_ASSET).approve(address(TRANSFER_STRATEGY), TOTAL_DISTRIBUTION);
+
+    IEmissionManager(AaveV3Ethereum.EMISSION_MANAGER).configureAssets(_getAssetConfigs());
+
+    emit log_named_bytes(
+      'calldata to submit from Gnosis Safe',
+      abi.encodeWithSelector(
+        IEmissionManager(AaveV3Ethereum.EMISSION_MANAGER).configureAssets.selector,
+        _getAssetConfigs()
+      )
+    );
+
+    vm.stopPrank();
+
+    // Not needed for this LM as ACI is first provider in this instance
+
+    // vm.startPrank(wETHLIDO_WHALE);
+    // IERC20(REWARD_ASSET).transfer(EMISSION_ADMIN, TOTAL_DISTRIBUTION); 
+    // vm.stopPrank();
+
+    _testClaimRewardsForWhale(WETH_A_TOKEN_WHALE, wETHLIDO_A_Token, 0.1 ether);
+  }
+
+function test_extendDistributionEnd() public {
+    // Initial setup
+    test_activation();
```

Comment:
not needed and should be removed from here as it updates block.timestamp, as inside `test_activation` when `_testClaimRewardsForWhale` is called: `vm.warp(block.timestamp + 14 days)` this will move the current timestamp to 14 days ahead so the current timestamp becomes not correct